### PR TITLE
fix: use correct manage plus link

### PR DESCRIPTION
**Pull request by omBratteng**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-737

### Preview domain
https://as-737-manage-plus.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/lib/constants.ts [hunk 1]
```diff
@@ -59,7 +59,8 @@ export const isChrome = (): boolean =>
 export const webappUrl = process.env.NEXT_PUBLIC_WEBAPP_URL;
 export const onboardingUrl = `${webappUrl}onboarding`;
 export const plusUrl = `${webappUrl}plus`;
-export const managePlusUrl = `${webappUrl}manage-plus`;
```
Comment (omBratteng) [inline packages/shared/src/lib/constants.ts:62]: Maybe we want this as a link via our redirector?

Comment (rebelchris) [reply packages/shared/src/lib/constants.ts:62]: Yeah though about it as well, let me ask Nimrod

**Review by rebelchris — APPROVED** [review]
For now it's fine

**Review by rebelchris — APPROVED** [review]
We should fix the test cases on the ad one though

**Pull request merged**